frontend/core-ui/src/modules/automations/hooks/useReacFlowEditor.tsx

```
@@ -0,0 +1,295 @@
+        if (visited.has(node.id)) return false;
+
+        visited.add(node.id);
+
+        for (const outgoer of getOutgoers(node, nodes, edges)) {
+          if (outgoer.id === connection.source) return true;
+          if (hasCycle(outgoer, visited)) return true;
+        }
+      };
+
+      if (target?.parentId && connection?.source && target.id) {
+      }
```

ESLint

[error] Empty block statement.

(no-empty)